pkg/kv/kvserver/metrics.go

```diff
@@ -2432,6 +2432,12 @@ Note that the measurement does not include the duration for replicating the eval
 		Measurement: "Nanoseconds",
 		Unit:        metric.Unit_NANOSECONDS,
 	}
+	metaAbsoluteKeyMovementCount = metric.Metadata{
```

Comment:
This metric tracks when key accesses seem to be moving in an absolute direction. It's similar to the popular key metric in that it aims to surface skewed data access within the cluster.

It also similarly will only show up when the replica deciders are engaged and sampling for split keys.